build/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: build
-version: 1.3.0
+version: 1.4.0-dev
```

Comment:
I think we can go ahead and remove the `-dev` here, we can publish once this is landed.

Reply:
I also removed the `-dev` suffix in the `build_resolvers` package.